### PR TITLE
doc: Enable warning as error on multiversion

### LINKED ISSUE #349: stable docs missing REST API usage
going to gnocchi.xyz, if looking at stable branches (http://gnocchi.xyz/stable_4.0/index.html, http://gnocchi.xyz/stable_3.1/index.html), the REST API Usage option is missing from table of contents but it can be viewed if you jump to link directly http://gnocchi.xyz/stable_3.1/rest.html

it also says for some reason:

> Warning: This document is for the development version of Gnocchi. The main version is stable/4.0.

similar to master

### PR DESCRIPTION
**Pull request by sileht**
Closes #349

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 2c84b863b0618c9a712f84b30135d06703a92104

**Comment by sileht** [comment]
Pull-request updated, HEAD is now bf2ef4fb0295a0f002ab41f3e089dbc6862e227c

**Comment by sileht** [comment]
Pull-request updated, HEAD is now f18ec9e6494c2a1c1e14c6771cdcac58431f9e88

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 4550baf4042b26c7eb9ffe55b5c72e5938a8c9b2

**Comment by sileht** [comment]
Pull-request updated, HEAD is now f98f79437279fe29d6609a0197683043a6d9b203

File: tox.ini [hunk 1]
```diff
@@ -152,6 +154,7 @@ deps = {[testenv:docs]deps}
        retrying
 commands =
     pifpaf -g GNOCCHI_INDEXER_URL run postgresql -- sphinx-versioning build doc/source doc/build/html
+    bash tools/validate_docs.sh
```
Comment (jd) [inline tox.ini]: s/bash//

File: gnocchi/gendoc.py [hunk 2]
```diff
@@ -99,28 +99,37 @@ def __getitem__(self, key):
...
-if version not in ["<local>", "master"] and os.path.exists("releasenotes"):
-    os.rename("doc/source/releasenotes/index.rst",
-        "doc/source/releasenotes/index.rst.backup")
-    with open("doc/source/releasenotes/index.rst", "w") as f:
-        f.write(\"\"\"
+if version not in ["<local>", "master"]:
+    # NOTE(sileht): Update _static files (mainly logos)
+    if not os.path.exists("doc/source/_static"):
+        os.makedirs("doc/source/_static")
+    for f in ("doc/source/_static/gnocchi-icon.ico",
```
Comment (chungg) [inline gnocchi/gendoc.py]: is it better if we discover what's in this folder?

Comment (sileht) [reply gnocchi/gendoc.py]: The idea is to only have the latest version of the logo copied everywhere. But currently if I copy the whole directory it should work.

**Pull request merged**